.github/workflows/hive.yml

```diff
@@ -139,46 +142,46 @@ jobs:
               - eth_getTransactionReceipt
               - eth_sendRawTransaction
               - eth_syncing
-          # debug_ rpc methods
-          - sim: ethereum/rpc-compat
-            include: [debug_]
+              # debug_ rpc methods
+              - debug_
           # Pyspec cancun jobs
-          - sim: pyspec
-            include: [cancun/eip4844]
-          - sim: pyspec
-            include: [cancun/eip4788]
-          - sim: pyspec
-            include: [cancun/eip6780]
-          - sim: pyspec
-            include: [cancun/eip5656]
-          - sim: pyspec
-            include: [cancun/eip1153]
-          - sim: pyspec
-            include: [cancun/eip7516]
+          # TODO: uncomment when https://github.com/ethereum/hive/issues/1147 is fixed
```

Comment:
let's create a tracking issue for this if you haven't already

Reply:
sure added here https://github.com/paradigmxyz/reth/issues/10427